index.html

```diff
@@ -29,17 +29,159 @@
           <li><img id="ellipse-04" src="elements/Ellipse-4.png" alt="icon-6"></li>
         </ul>
         
-        <div id="wrapper">
-            <h1 id="primary-text">I'm Enmanuel, glad to see you!</h1>
-            <p id="secondary-text">Im a software!, I can help you build a product, feature or website look through some of my work and experience! 
+        <div class="wrapper">
+            <h2 class="primary-text">I'm Enmanuel, glad to see you!</h2>
+            <p class="secondary-text">Im a software!, I can help you build a product, feature or website look through some of my work and experience! 
                 If you like what you see and have a project you need coded,don't hesitate to contact me.</p>
-            <p id="third-text">let's connect</p>
+            <p class="third-text">let's connect</p>
             <div id="social">
-                <img src="elements/twitter-icon.png" alt="">
-                <img src="elements/github-icon.jpg" alt="">
+                <img src="elements/twitter-icon.png" alt="twitter-icon">
+                <img src="elements/github-icon.jpg" alt="github-icon">
                 
             </div>
         </div>
     </main>
+
+    <ul class="work">
+        <li class="project">
+            <div class="screenshot">
+                <img src="elements/snapshoot.png" alt="snapshoot-1">
+            </div>
+            <h2 class="project-name">tonic</h2>
+            <div class="meta">
+                <p>canopy</p>
+                <img src="elements/counter.png" alt="">
+                <p>back end dev</p>
+                <img src="elements/counter.png" alt="">
+                <p>2015</p>
+            </div>
+            <p class="second-text">A daily selection of privately personalized reads;
+                   no accounts or sign-ups required.
+            </p>
```

Comment:
- [ ] Well done 🥳 on making the requested change on this part. However, was it necessary to remove the container `<section>` tag! Kindly keep in mind that according to the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) each section of the page should be wrapped inside a `<section>` tag. Currently, you're making the whole section is a `<ul>` tag (which is awesome work 💯) Now, let's wrap the `<ul>` inside our `<section>` with considering the styling of the container such as the `disply: grid`.